packages/collector/test/tracing/database/ioredis/test.js

```diff
@@ -1281,4 +1281,137 @@ function checkConnection(span, setupType) {
       });
     }
   });
+  mochaSuiteFn('ignore-endpoints test:', function () {
+    this.timeout(config.getTestTimeout() * 4);
+    describe('ignore-endpoints enabled via agent config', () => {
+      const { AgentStubControls } = require('../../../apps/agentStubControls');
+      const customAgentControls = new AgentStubControls();
+      let ignoreControls;
+
+      before(async () => {
+        await customAgentControls.startAgent({
+          ignoreEndpoints: { redis: 'type|set' }
+        });
+
+        ignoreControls = new ProcessControls({
+          agentControls: customAgentControls,
+          dirname: __dirname,
+          env: {
+            REDIS_CLUSTER: setupType === 'cluster'
+          }
+        });
+        await ignoreControls.startAndWaitForAgentConnection(5000, Date.now() + 1000 * 60 * 5);
+      });
+
+      beforeEach(async () => {
+        await customAgentControls.clearReceivedTraceData();
+      });
+
+      after(async () => {
+        await customAgentControls.stopAgent();
+        await ignoreControls.stop();
+      });
+
+      it('should ignore redis spans for configured ignore endpoints', async () => {
+        await ignoreControls.sendRequest({
+          method: 'POST',
+          path: '/values',
+          qs: {
+            key: 'discount',
+            value: 50
+          }
+        });
+
+        const spans = await customAgentControls.getSpans();
+        spans.forEach(span => {
+          expect(span.n).not.to.equal('redis');
+        });
+      });
+    });
+    describe('ignore-endpoints enabled via tracing config', async () => {
+      globalAgent.setUpCleanUpHooks();
+      const agentControls = globalAgent.instance;
+      let ignoreControls;
+
+      before(async () => {
+        ignoreControls = new ProcessControls({
+          useGlobalAgent: true,
+          dirname: __dirname,
+          env: {
+            REDIS_CLUSTER: setupType === 'cluster',
+            IGNORE_ENDPOINTS: true,
+            IGNORE_COMMANDS: JSON.stringify(['get'])
```

Comment:
qs: What is this "IGNORE_ENDPOINTS" and "IGNORE_COMMANDS"?

Reply:
removed IGNORE_COMMANDS